pkgs/development/ocaml-modules/ezgzip/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  alcotest,
+  astring,
+  buildDunePackage,
+  camlzip,
+  fetchFromGitHub,
+  lib,
+  ocplib-endian,
+  qcheck,
+  rresult,
+}:
+
+buildDunePackage (finalAttrs: {
+  pname = "ezgzip";
+  version = "0.2.3";
+  minimalOCamlVersion = "4.03.0";
+  src = fetchFromGitHub {
+    owner = "hcarty";
+    repo = "ezgzip";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-OQ4JT1pYkeJbi8iMGpcFp8j0DawZCguFfWQmJCwgUXQ=";
+  };
+  propagatedBuildInputs = [
```

Comment:
Please download the proper release artifact from https://github.com/hcarty/ezgzip/releases/download/v0.2.3/ezgzip-v0.2.3.tbz

Reply:
Will change that.

Just out of curiosity. What is the advantage to using that release artifact over fetchFromGitHub?